src/app/pages/storage/volumes/datasets/dataset-unlock/dataset-unlock.component.ts

```diff
@@ -160,6 +164,31 @@ export class DatasetUnlockComponent implements FormConfiguration {
       name: 'encrypted_roots_divider',
       divider: true,
     },
+    {
+      name: helptext.restart_services_placeholder,
```

Comment:
The placeholder is set to the name attribute. Name should be like a variable name or id without white spaces etc.

Reply:
In this case, `name` is used as a section label.
![image](https://user-images.githubusercontent.com/351613/122355707-cab81500-cf5a-11eb-8b21-e6a7ab1e92ea.png)